bitnami/redis/Chart.yaml

```diff
@@ -25,4 +25,4 @@ name: redis
 sources:
   - https://github.com/bitnami/bitnami-docker-redis
   - http://redis.io/
-version: 14.1.1
+version: 14.1.2
```

Comment:
Due to the amount of changes, I would suggest to increase the minor revision

Reply:
done